### PR TITLE
board columns height modified

### PR DESCRIPTION
**Pull request by babalwa-mbolekwa**
Related issues: [https://github.com/Umuzi-org/Tilde/issues/156]

## Description:

What are you up to? Fill us in :)

## Screenshots/videos
boardColumns

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This doesn't work. Try zooming in and out.

It needs to work on all screen sizes. Not just your personal screen size.  Where did you get 79vh from?  Where did you get 93% from?

Take a look here. I zoomed out quite a lot to make the problem obvious/. That blank space at the bottom should not be blank. The columns should reach the bottom of the screen
Screenshot from 2021-09-02 07-19-13

**Comment by babalwa-mbolekwa** [comment]
zoomOut
zoomIn
fullView

File: frontend/src/components/regions/AgileBoard/Presentation.js [hunk 1]
```diff
@@ -23,8 +23,12 @@ const useStyles = makeStyles((theme) => ({
     },
   },
 
+  paper: {
+    height: "calc(100vh - 9rem)",
```
Comment (sheenarbw) [inline frontend/src/components/regions/AgileBoard/Presentation.js]: probably 6rem is more appropriate than 9 rem. Ideally the gap at the top should match the gap at the bottom. We want things to be tidy.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
You got it to work, but it's a bit untidy because the spacing doesn't match up

**Comment by sheenarbw** [comment]
merged on my computer.

**Pull request closed without merging**